pkg/ddc/alluxio/health_check.go

```diff
@@ -165,17 +167,17 @@ func (e *AlluxioEngine) checkWorkersHealthy() (err error) {
 
 		runtimeToUpdate := runtime.DeepCopy()
 
-		if workers.Status.NumberUnavailable > 0 ||
-			(workers.Status.DesiredNumberScheduled > 0 && workers.Status.NumberAvailable == 0) {
+		if *workers.Spec.Replicas-workers.Status.ReadyReplicas > 0 ||
```

Comment:
I think the meaning of not ready is that if `readyReplicas==0 && *workers.spec.replicas > 0`

Reply:
thank you so much for your help ,otherwise may produce a bug...